app/helpers/application_helper.rb

```diff
@@ -70,4 +71,14 @@ def flash_message(name, msg)
     return sanitize(msg) if name.end_with? "html"
     msg
   end
+
+  private
+
+  def default_avatar(theme:)
+    case theme
+    when :light then "/images/avatar.svg"
+    when :dark then "/images/avatar_inverted.svg"
```

Comment:
I've often seen SVGs styled directly in css to change the fill color. Is that worth it here? Something like [this](https://stackoverflow.com/questions/22252472/how-can-i-change-the-color-of-an-svg-element/53336754#53336754) with invert() or even inlining the svg so it doesn't take an extra request.

Reply:
I think doing it in the CSS makes sense, then the SVG will change without a page refresh

Reply:
This is not related to dark mode, but I'll try to provide inline SVG with styles, that could be possible.

Reply:
I was tired when I wrote this. Don't `invert()` it unless we want emerald users, haha.